manifests/kustomize/README.md

```diff
@@ -48,7 +48,30 @@ kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
 ```
 and open http://localhost:8080/
 
+### Deploy on AWS (with S3 buckets as artifact store)
+
+[https://github.com/e2fyi/kubeflow-aws](https://github.com/e2fyi/kubeflow-aws/tree/master/pipelines)
+provides a community-maintained manifest for deploying kubeflow pipelines on AWS
+(with S3 as artifact store instead of minio).
+
+TL;DR
```

Comment:
Shall we also move these instructions into the new repo?
I think that makes it more self contained, and instruction + manifests are versioned together makes more sense

Reply:
We should probably still have at least a small section about AWS and a pointer to the other repo. @IronPan WDYT?

Reply:
Agree, I was thinking the section only introduces the repo.